components/ContainedGrid.js

```diff
@@ -1,8 +1,12 @@
 import React from 'react'
 import PropTypes from 'prop-types'
 
-const ContainedGrid = ({ children }) => (
-  <div className="container">
+const ContainedGrid = ({
+  children,
+  className,
+  withoutContainer,
+}) => (
+  <div className={`${className} ${withoutContainer ? '' : 'container'}`}>
```

Comment:
Isn't the whole purpose of something called `ContainedGrid` to add a container?

Reply:
Yeah, I was thinking about the naming. But I couldn't come up with something reasonable.

Reply:
I will rename it to `CenteredGrid`